cli/server.go

```diff
@@ -183,7 +183,9 @@ func server() *cobra.Command {
 					"Coder requires a URL accessible by workspaces you provision. "+
 						"A free tunnel can be created for simple setup. This will "+
 						"expose your Coder deployment to a publicly accessible URL. "+
-						cliui.Styles.Field.Render("--access-url")+" can be specified instead.\n",
+						"If you'd like to use your own domain, you can provide it "+
```

Comment:
```suggestion
						"If you'd like to use your own hostname, you can provide it "+
```

`hostname` includes IP addresses.